scripts/endpoints-checker.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+set -uo pipefail
+${TRACE:+set -x}
+
+CHAIN_FILE=$1
+THRESHOLD=${THRESHOLD:-30}
+TIMEOUT=${TIMEOUT:-5}
+
+is_synced() {
+    local block_time=$1
+
+    if [[ -z $block_time ]]; then
```

Comment:
I don't think this is needed since `jq` returns `null` always if there's no value in json

Reply:
I'm using bash flag `-o pipefail` to have proper exit code when curl fails so jq won't be executed and `block_time` will be empty in this case. This is the check here.